### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by Hwesta**
Add Contributor Covenant Code of Conduct v1.4 from http://contributor-covenant.org/version/1/4/

This should also be added to our other projects (eg storage service, devtools, automation tools etc)

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. The project team
```
Comment (Hwesta) [inline CODE_OF_CONDUCT.md:58]: What email address do we want here?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
```
Comment (mcantelon) [inline CODE_OF_CONDUCT.md:10]: "political affiliation" and "socioeconomic class" could be good to add as well.

Comment (Hwesta) [inline CODE_OF_CONDUCT.md:10]: There's a PR on the original repo about class, actually.  https://github.com/ContributorCovenant/contributor_covenant/pull/82 Maybe you can make a PR for political affiliation?

Comment (mcantelon) [inline CODE_OF_CONDUCT.md:10]: Nice... will do.

Comment (Hwesta) [inline CODE_OF_CONDUCT.md:10]: Looks like politics is discussed here https://github.com/ContributorCovenant/contributor_covenant/issues/73 and was rejected.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (sallain) [inline CODE_OF_CONDUCT.md:58]: I'd recommend info@, but with added caveat that this email is sent to many people in the office to ensure maximum coverage. Especially in Codes of Conduct, I think it's important to establish a direct line to people who are able to take action.

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (mcantelon) [inline CODE_OF_CONDUCT.md:10]: The rejection seems a bit arbitrary, but que sera sera. Weird that the class PR's in limbo... that seems straightforward (although discrimination based on class tends to be subtler than other forms).

**Comment by sromkey** [comment]
I'd prefer to discuss this after the 1.7 release is done.

**Comment by jhsimpson** [comment]
This Code of Conduct is being added to a new Issues repo in the new Archivematica githbub repo here: https://github.com/archivematica/Issues/pull/6

I am closing this PR in favour of that one.

**Pull request closed without merging**